web/viewer.js

```diff
@@ -1311,12 +1340,30 @@ var PDFViewerApplication = {
 //#endif
   },
 
+  // Whether all pages of the PDF have the same width and height.
+  hasEqualPageSizes: function pdfViewHasEqualPageSizes() {
```

Comment:
I'd suggest changing this to a getter instead, since you're not passing any parameters to the function and it only returns a boolean.